config/formats.js

```diff
@@ -226,6 +226,32 @@ exports.Formats = [
 			}
 		},
 	},
+	{
+		name: "[Gen 7] Ultra Sinnoh Classic",
+		desc: [
+			'&bullet; <a href="http://www.smogon.com/forums/threads/3627404/">Ultra Sinnoh Classic Discussion</a>',
```

Comment:
This should be surrounded by ``` ` ``` quotes, not `'` ones.
`desc` can all be on one line because there's only one link:
```javascript
desc: [`&bullet; <a href="http://www.smogon.com/forums/threads/3627404/">Ultra Sinnoh Classic Discussion</a>`],
```